hugo/data/teams.yml

```diff
@@ -115,13 +112,20 @@ sections:
           github: https://github.com/m09
         - picture: konstantin.png
           name: Konstantin Slavnov
-          position: Junior Software Engineer
+          position: Software Engineer
           github: https://github.com/zurk
         - picture: waren.png
           name: Waren Long
           position: Junior Software Engineer
```

Comment:
I read that Waren was also promoted as Software Engineer.

Reply:
Thanks! Done.